db/blip_handler_test.go

```diff
@@ -18,16 +18,27 @@ import (
 )
 
 func TestCollectionBlipHandler(t *testing.T) {
-	allDBContext := &DatabaseContext{}
-	allDBCollection := &DatabaseCollection{dbCtx: allDBContext}
+	allDBContext := &DatabaseContext{Name: "db"}
+	ctx := base.TestCtx(t)
+	dbStats, err := initDatabaseStats(ctx, allDBContext.Name, false, DatabaseContextOptions{})
+	require.NoError(t, err)
+	allDBContext.DbStats = dbStats
+	bucket := base.GetTestBucket(t)
+	defer bucket.Close()
+	allDBCollection, err := newDatabaseCollection(ctx, allDBContext, bucket.GetSingleDataStore(), nil)
+	require.NoError(t, err)
 	allDBContext.CollectionByID = map[uint32]*DatabaseCollection{
 		base.DefaultCollectionID: allDBCollection,
 	}
 	allDB := &Database{
 		DatabaseContext: allDBContext,
 	}
-	realCollectionDB0 := &DatabaseCollection{dbCtx: &DatabaseContext{}}
-	realCollectionDB1 := &DatabaseCollection{dbCtx: &DatabaseContext{}}
+	allDBContext.Bucket = bucket
+	require.NoError(t, err)
+	realCollectionDB0, err := newDatabaseCollection(ctx, allDBContext, bucket.GetSingleDataStore(), nil)
+	require.NoError(t, err)
+	realCollectionDB1, err := newDatabaseCollection(ctx, allDBContext, bucket.GetSingleDataStore(), nil)
+	require.NoError(t, err)
```

Comment:
This change isn't in `master` - is it only appliccable to the `3.1.x` branch? Or is this something that needs a new PR into master?

https://github.com/couchbase/sync_gateway/blame/2da8f9c31b3f33fdc114509e9204454fd1c6cb11/db/blip_handler_test.go#L20-L30

Reply:
This is only relevant to 3.1 branch, we changed this in https://github.com/couchbase/sync_gateway/commit/6dee30f21e68eb6fc26aafdff1aba46a006a7613